rom/dev/src/lock.rs

```diff
@@ -35,9 +35,10 @@ pub fn lock_registers(env: &mut RomEnv, reset_reason: ResetReason) {
         lock_common_reg_set(env);
     }
 
-    // Lock the PCR0 from clear
-    cprintln!("[state] Locking PCR0");
+    // Lock PCR0 and PCR1 from clear
+    cprintln!("[state] Locking PCR0 and PCR1");
     env.pcr_bank.set_pcr_lock(caliptra_drivers::PcrId::PcrId0);
```

Comment:
We should describe the static allocation scheme of pcrs same way you did for key slots.

Reply:
I.e. use named constants here, defined in a central file under `common/`? Works for me, can do as a follow-up (just looking to avoid adding more moving parts to this PR).

Reply:
Sure.